tests/test_standalone_app.py

```diff
@@ -149,9 +145,9 @@ def test_open_script(console_widget, current_path, script):
 @pytest.mark.parametrize(
     "script, python_version, expected_output",
     [
-        ("resource_script.py", 2, "open script"),
+        ("resource_script.py", 3, "open script"),
         # Check that it handles error in an expected way.
-        ("resource_script_error.py", 2, "NameError: name 'b' is not defined"),
+        ("resource_script_error.py", 3, "NameError: name 'b' is not defined"),
```

Comment:
I think this needs a bit more refactoring since the second parameter (3) indicates a Python major version and we only have 3 ...

Reply:
I was considering something like this `("resource_script.py", (3, 11), "open script")`, but it looks more simplified and legible without the python_version validation (like in my last [commit](https://github.com/shotgunsoftware/tk-multi-pythonconsole/pull/38/commits/bc6ee63c73188d4ca3c05fcac8f3e29b546843a5)), since, as you mentioned, we only have Python 3.